core/src/main/scala/org/apache/spark/ErrorInfo.scala

```diff
@@ -58,7 +58,8 @@ private[spark] object SparkThrowableHelper {
   def getMessage(errorClass: String, messageParameters: Array[String]): String = {
     val errorInfo = errorClassToInfoMap.getOrElse(errorClass,
       throw new IllegalArgumentException(s"Cannot find error class '$errorClass'"))
-    String.format(errorInfo.messageFormat.replaceAll("<[a-zA-Z0-9_-]+>", "%s"),
+    "[" + errorClass + "] " + String.format(
```

Comment:
maybe:

```suggestion
    s"[$errorClass] " + String.format(
```

Reply:
Let's do this later. I guess we will touch the method one more time at least in the future.